composer.json

```diff
@@ -25,44 +25,44 @@
         "symfony/asset": "self.version",
         "symfony/browser-kit": "self.version",
         "symfony/class-loader": "self.version",
-        "symfony/config": "2.99.99",
-        "symfony/console": "2.99.99",
+        "symfony/config": "self.version",
+        "symfony/console": "self.version",
         "symfony/css-selector": "self.version",
-        "symfony/dependency-injection": "2.99.99",
+        "symfony/dependency-injection": "self.version",
         "symfony/debug": "self.version",
         "symfony/debug-bundle": "self.version",
-        "symfony/doctrine-bridge": "2.99.99",
+        "symfony/doctrine-bridge": "self.version",
```

Comment:
This change is blocked until https://github.com/doctrine/DoctrineCacheBundle/pull/48 is merged